src/layers/core/arc-layer/arc-layer.js

```diff
@@ -42,17 +42,10 @@ const defaultProps = {
 
 export default class ArcLayer extends Layer {
   getShaders() {
-    return enable64bitSupport(this.props) ? {
-      vs: arcVertex64,
-      fs: arcFragment,
-      modules: ['fp64', 'project64'],
-      shaderCache: this.context.shaderCache
-    } : {
-      vs: arcVertex,
-      fs: arcFragment,
-      modules: [],
-      shaderCache: this.context.shaderCache
-    };
+    const {shaderCache} = this.context;
+    return enable64bitSupport(this.props) ?
+      {vs: vs64, fs, modules: ['project64'], shaderCache} :
```

Comment:
There are no default modules anymore. Instead, shader modules now can specify `dependencies` on other modules and the luma.gl `assembleShaders` automatically resolves the resulting DAG (directed acyclic graph) and injects dependencies recursively and deduplicated in the right order.